### ISSUE TITLE
多分类实现的困惑

### ISSUE THREAD

**Issue by stupidHIGH** [shenweichen/DeepCTR#166]
Please refer to the [FAQ](https://deepctr-doc.readthedocs.io/en/latest/FAQ.html) in doc and search for the [related issues](https://github.com/shenweichen/DeepCTR/issues) before you ask the question.

**Describe the question(问题描述)**

你好，我在自己实现多分类的时候遇到一些问题请教一下作者该怎么正确修改。
1、deepfm.py文件中定义dnn_logit = tf.keras.layers.Dense(1, use_bias=False, activation=None)(dnn_out)的那一行输出神经元个数需要修改吗，如果是，修改为多少呢？
2、core.py文件 PredictionLayer对象中方法call()定义了当task="binary"时使用x = tf.sigmoid(x)，当我添加当task="multiclass"时，我添加了x=tf.nn.sofmax(x)，但针对二分类的output = tf.reshape(x, (-1, 1))语句该修改为多少呢？
希望问题可以得到解答，不胜感激，谢谢！！
**Additional context**

**Operating environment(运行环境):**
 - python version [e.g. 3.6]
 - tensorflow version [e.g. 1.4.0,]
 - deepctr version [e.g. 0.5.2,]

**Comment by shenweichen** [comment]
修改为多分类的类别个数即可
